src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudRestartPreemptedIT.java

```diff
@@ -66,8 +65,11 @@
  */
 @Log
 public class ComputeEngineCloudRestartPreemptedIT {
-    @ClassRule
-    public static Timeout timeout = new Timeout(20 * TEST_TIMEOUT_MULTIPLIER, TimeUnit.MINUTES);
```

Comment:
Using the `Timeout` wasn't working, the JenkinsRule was still timing out at 180s, so fixing it with setting a property

Reply:
add `@WithTimeout(1200)` at test level.